packages/vite/src/node/plugins/css.ts

```diff
@@ -404,7 +404,7 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
         }
 
         if (isDirectCSSRequest(id)) {
-          return await getContentWithSourcemap(css)
+          return htmlProxyRE.test(id) ? await getContentWithSourcemap(css) : css
```

Comment:
I think we should generate a sourcemap here.
https://github.com/vitejs/vite/blob/a8c7eb24a2b7d8dc765d59758e2c6b930eb90af1/packages/vite/src/node/plugins/html.ts#L101
Then, we can just run `return null` in this line I guess.

Reply:
I'm not sure to understand. I've read part of the code of the plugin and I don't understand where is the pipeline called. I've the impression that the cache map is set with the tag content directly without transformation and then on this line we return this content

Reply:
Ah, my bad. I was misunderstanding the code. I pushed the code I had in my mind.

Reply:
Oh ok I didn't saw that call to the pluginContainer and just saw the load part.
I have no opinion on where this code 🤷‍♂️